lhotse/bin/lhotse

```diff
@@ -1,12 +1,12 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 """
 Use this script like:
 
 $ lhotse --help
 $ lhotse make-feats --help
 $ lhotse make-feats --compressed recording_manifest.yml mfcc_dir/
 $ lhotse write-default-feature-config feat-conf.yml
-$ lhotse convert-kaldi data/train/ 16000 train_manifests/
+$ lhotse kaldi import  data/train 16000 train_manifests/
```

Comment:
Nit: There is an extra space after `import`.

Reply:
Done!